pkg/qbert/qbert.go

```diff
@@ -387,6 +388,42 @@ func (c QbertImpl) CheckClusterExists(name, projectID, token string) (bool, stri
 	return false, "", nil
 }
 
+func (c QbertImpl) CheckClusterExistsWithUuid(uuid, projectID, token string) (bool, string, error) {
+	qbertApiClustersEndpoint := fmt.Sprintf("%s/qbert/v3/%s/clusters", c.fqdn, projectID) // Context should return projectID,make changes to keystoneAuth.
+	client := http.Client{}
+	req, err := http.NewRequest("GET", qbertApiClustersEndpoint, nil)
+
+	if err != nil {
+		return false, "", fmt.Errorf("Unable to create request to check cluster name: %w", err)
+	}
+
+	req.Header.Set("X-Auth-Token", token)
+	req.Header.Set("Content-Type", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		return false, "", err
+	}
+	if resp.StatusCode != 200 {
+		return false, "", fmt.Errorf("Couldn't query the qbert Endpoint: %d", resp.StatusCode)
```

Comment:
* Errors should not be capitalized
* Avoid colloquialisms
```suggestion
		return false, "", fmt.Errorf("could not query the qbert Endpoint: %d", resp.StatusCode)
```
(same goes for other errors)